src/main/java/Deadline.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+
+public class Deadline extends Task {
+    String due;
+    String parsedDue;
+
+    /**
+     * Creates deadline type.
+     *
+     * @param description Describes the task.
+     */
+    public Deadline(String[] description) {
+        super(description[0]);
+        due = (description[1].split(" ", 2))[1];
```

Comment:
Hello! The checkstyle format for this Deadline.java is good! I think it will be better if you passed all the .splitting to the Parser class. Apart from that its good!